code/public/index.html

```diff
@@ -4,6 +4,9 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"/>
+
```

Comment:
I always forgot about the favicon, but I love that you are paying attention for these small things!(: